app/models/relationship.rb

```diff
@@ -0,0 +1,29 @@
+class Relationship < Term 
+
+  attr_accessor :commit_history
+
+  configure :type => RDF::URI("http://vivoweb.org/ontology/core#Relationship")
+
+  property :parents, :predicate => RDF::Vocab::SKOS.broader
```

Comment:
So parent is a keyword and is un-usable. It throws an error when running. Is there another name that you would prefer?